packages/polyfills/src/index.ts

```diff
@@ -32,7 +32,7 @@ export const polyfills: {[polyfill: string]: PolyfillDescriptor} = {
 export function mappedPolyfillsForEnv(
   env: 'node' | 'jest' | string[],
 ): {[key: string]: string} {
-  const prefix = `@shopify/polyfills/dist/src`;
+  const prefix = `@shopify/polyfills/build/ts`;
```

Comment:
I defaulted this to `build/ts` to match our previous behaviour with `dist/src`. However, this probably means we won't need to include the other build types for this package? I don't see how consumers would access them, unless we rework this package to add an entrypoint for each polyfill?

Reply:
**Update:** changed this to `cjs` since we shouldn't be looking into the `ts` builds. 

**Question:** should we have multiple builds/entrypoints for this package and have this function map to the entrypoints? Or should we have it map to just the CommonJS builds? I feel like there's not a lot of value in trying to make it treeshakeable since `sideEffects` is `true` anyway, and most of the files are only a few lines at most, but maybe someone with more context can weigh in @lemonmade @ismail-syed